sdk/batch/azure-compute-batch/src/main/java/com/azure/compute/batch/BatchAsyncClient.java

```diff
@@ -10809,14 +12289,8 @@ public Mono<BatchApplication> getApplication(
      * @return contains information about an application in an Azure Batch Account on successful completion of {@link
      *     Mono}.
      */
-    @Generated
-    @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<BatchApplication> getApplication(String applicationId) {
-        // Generated convenience method for getApplicationWithResponse
-        RequestOptions requestOptions = new RequestOptions();
-        return getApplicationWithResponse(applicationId, requestOptions)
-                .flatMap(FluxUtil::toMono)
-                .map(protocolMethodData -> protocolMethodData.toObject(BatchApplication.class));
+    public Mono<BatchApplication> getApplication(String applicationId, OptionsBag optionsBag) {
```

Comment:
This is a good example of options bags not being the same for each operation. In this case, only timeOutInSeconds is valid, but not any of the ones for lists like filter, maxResults, etc.